reference/platform-app-yaml.md

```diff
@@ -228,7 +228,7 @@ application in MB.
 ### Mounts
 
 The `mounts` is an object whose keys are paths relative to the root of
-the application. It's in the form `volume_id[/subpath]`.
+the application (That is, where the `.platform.app.yaml` file lives). It's in the form `volume_id[/subpath]`.
```

Comment:
For consistency, shall we use the following?

> (That is, where the `.platform.app.yaml` file is located).

See description of `root` above.